Description: Set HTML_TIMESTAMP from YES to NO to make package reproducible

--- a/Doxyfile.in
+++ b/Doxyfile.in
@@ -1240,7 +1240,7 @@ HTML_COLORSTYLE_GAMMA  = 80
 # The default value is: NO.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_TIMESTAMP         = YES
+HTML_TIMESTAMP         = NO
 
 # If the HTML_DYNAMIC_MENUS tag is set to YES then the generated HTML
 # documentation will contain a main index with vertical navigation menus that
